CHANGELOG.asciidoc

```diff
@@ -18,6 +18,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 *Auditbeat*
 
 - Use `initial_scan` action for new paths. {pull}7954[7954]
+- Rename beat.name to beat.type, beat.hostname to agent.hostname, beat.version to agent.version.
```

Comment:
Should "Rename beat.name to beat.type" read "Rename beat.name to **agent**.type"?

Reply:
Fixed, thanks.